[SDPA-2718] Update docs / new-component / package-dependencies scripts.

alan-cole opened the pull request:
<!--
Please follow these rules:
1. SUBJECT: use format [SDPA-123] Verb in past tense with dot at the end.
   - This subject will be used as a commit message after PR is merged.
   - Verbs are usually one of these: Updated, Refactored, Removed, Changed, Added.
   - If there is no ticket - do not put [NOTICKET].

2. BODY: fill-in the template below

3. LABEL: Assign 'Needs review' label as soon as you ready to have this reviewed.

4. ASSIGNEE: Assign at least 2 reviewers.     

5. SLACK: Post a link to this PR to #developers channel.

No need to remove these lines - they are comments.
-->

**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-2718

### Changed

* Add scripts to root package.
* Update scripts with 'yarn' naming and updated paths.
* Replace 'file://' with version in ripple-ui-components/package.json.
* Change 'new-package' naming to 'new-component' to match dir naming.
* Add next steps prompt after new package is generated.
* Get versions from package.json rather than lerna.json.
* Update repository path on package.json template for new components.

### Screenshots

<!--
Provide as many screenshots as required to make reviewers understand what was changed.
-->
<img width="443" alt="Screen Shot 2019-07-04 at 1 44 39 pm" src="https://user-images.githubusercontent.com/12739575/60638504-f6bb5800-9e61-11e9-8f76-c5dcd4165bd2.png">
<img width="753" alt="Screen Shot 2019-07-04 at 1 44 26 pm" src="https://user-images.githubusercontent.com/12739575/60638506-f6bb5800-9e61-11e9-9511-3e7030385b7d.png">
<img width="581" alt="Screen Shot 2019-07-04 at 1 44 00 pm" src="https://user-images.githubusercontent.com/12739575/60638507-f753ee80-9e61-11e9-99a5-8e6871fd53c8.png">

tim-yao commented on packages/ripple-ui-components/scripts/generate-component/stories_template.js:
```diff
@@ -6,17 +6,17 @@ import {
   withKnobs
```
As storybook has been upgraded to 5, a lot changes happened. This is outdated. Please update it according to latest changes in storybook.

alan-cole commented:
Updated scripts to work with Storybook v5.

* Updates to stories template
* Updated URLs in package.json
* Updated readme.MD storybook links to support multiple .storiesOf() functions with add.
* Updated paths.
